### PR TITLE
handle SurveyDataTransferRequest with mocked callbacks

### PR DESCRIPTION
**Pull request by justinchou-google**
b/249326415
handle SurveyDataTransferRequest and return SurveyDataTransferResponse to the survey iframe.

### CODE REVIEW

File: src/runtime/audience-action-flow.js [hunk 1]
```diff
@@ -253,6 +259,20 @@ export class AudienceActionFlow {
...
   }
 
+  /**
+   * @param {SurveyDataTransferRequest} request
+   * @private
+   */
+  // eslint-disable-next-line no-unused-vars
+  handleSurveyDataTransferRequest_(request) {
+    // @TODO(justinchou): execute callback with
+    // setOnInterventionComplete and Google Analytics
```
Comment (oyj9109) [inline src/runtime/audience-action-flow.js:269]: Can you run linter to arrange the comment lines?

**Review by oyj9109 — APPROVED** [review]
LGTM!

File: src/runtime/audience-action-flow.js (hunk 1 above)
Comment (justinchou-google) [reply src/runtime/audience-action-flow.js]: not sure if the formatter is smart enough to fix all formatting issues (as formatting the doc isn't changing anything for me), but I think you're asking to condense them a bit more? see changes

Comment (oyj9109) [reply src/runtime/audience-action-flow.js]: Yeah, looks good. Thanks!

**Review by hartmannr76 — APPROVED** [review]

**Pull request merged**